config/locales/en.yml

```diff
@@ -299,8 +299,16 @@ en:
         label:
           filled?: "must be filled"
           not_unique: "must be unique within the same hierarchy level"
+        short:
+          not_unique: "must be unique within the same hierarchy level"
+        item:
+          root_item: "cannot be a root item"
+          not_persisted: "must be an already existing item"
     rules:
       label: "Label"
+      short: "Short"
+      item: "Item"
+      parent: "Item"
```

Comment:
🔴 This one will clash with my current PR. Why not translating key `parent` to `Parent`? And if it is a very specific translation (for this contract), then it should be below a namespace.

Reply:
I'm ok with that.